README.md

```diff
@@ -20,6 +20,8 @@ npm install -g @asyncapi/generator
 |Name|Description|Required|Default|Example|
 |---|---|---|---|---|
 |outFilename|The filename of the output file.|No|`asyncapi.md`|`index.md`|
+|version|Override the Async API version displayed in the generated pages.|No|*Any* ([See Semver versionning](https://semver.org/))|`1.0.0`|
```

Comment:
```suggestion
| version | Override the version of your application provided under `info.version` location in the spec file. | No | Version is taken from the spec file. |`1.0.0`|
```

Reply:
hi, is it normal that you suggest adding `Version is taken from the spec file.` into the `Allowed values` ?

Reply:
what `Allowed values` do you mean? I suggest it under `Default`

Reply:
in `html-template`, this is a `Allowed values` column

Reply:
done for the `markdown` side

Reply:
ah yes, we are not consistent yet, we set some standard here https://github.com/asyncapi/template-for-generator-templates/blob/master/_README.md#template-configuration just need to adjust, could you do this in this and the other PR? 🙏🏼

Reply:
yes, it is done